### PR TITLE
Pipeline runner fixes

### PR DESCRIPTION
**Pull request by hanars**
This makes improvements to the usability of our pipeline-runner docker image. Big changes:

- adds a missing set of python utilities (fixes user reported bug)
- adds VEP configurations for running in dataproc (fixes user reported bug)
- adds bash scrips for one-time downloading/ copying of reference data (instead of copy-pasting commands from the README)
- adds bash scripts for actually running the pipeline (instead of copy-pasting commands from the README)

This makes no changes to seqr itself, so it's release does not actually require a seqr release

- [x] All unit test are passing and coverage has not substantively dropped
- [x] No new dependency vulnerabilities have been introduced
- [x] Static assets have been built and committed if needed
- [x] If any python requirements are updated, they are noted and here and will be updated before deploying: 
- [x] Any database migrations are noted here, and will be run before deploying: 
- [x] All major changes are recorded in the changelog, and the changelog has been updated to reflect the latest release
- [x] Any new endpoints are explicitly tested to ensure they are only accessible to correctly permissioned users
- [x] No secrets have been committed
- [x] Infrastructure changes:
  - [ ] No changes to the required seqr infrastructure are included in this change 
  
  OR 
  
  - [x] Any changes to seqr's infrastructure are already been deployed to a new docker image, and these changes will be released via that image. All these changes have been tested before release on the docker image
  -  image is built but not pushed to registry, will push as soon as this is merged
  - [x] Any changes to kubernetes configurations will be deployed through a full seqr kubernetes deployment after merging. All these changes have been tested on dev
  - [x] All these changes have bee vetted for potential vulnerabilities

### CODE REVIEW

File: deploy/docker/pipeline-runner/bin/copy_reference_data_to_gs.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+DEST_BUCKET=${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}
+
+REF_DATA_HT=combined_reference_data_grch${BUILD_VERSION}.ht
+gsutil rsync -r gs://seqr-reference-data/GRCh${BUILD_VERSION}/all_reference_data/${REF_DATA_HT} ${DEST_BUCKET}/${REF_DATA_HT}
+
+CLINVAR_HT=clinvar.GRCh${BUILD_VERSION}.2021-11-13.ht
+gsutil rsync -r gs://seqr-reference-data/GRCh${BUILD_VERSION}/clinvar/${CLINVAR_HT} ${DEST_BUCKET}/${CLINVAR_HT}
+
+gsutil cp /vep_configs/vep-GRCh${BUILD_VERSION}-loftee-dataproc.json ${DEST_BUCKET}
```
Comment (sjahl) [inline deploy/docker/pipeline-runner/bin/copy_reference_data_to_gs.sh:16]: missing newline

File: deploy/docker/pipeline-runner/bin/load_data_dataproc.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+         --clinvar-ht-path ${REFERENCE_DATA_BUCKET}/clinvar.${FULL_BUILD_VERSION}.2021-11-13.ht
+
+JOB_ID=$(gcloud dataproc jobs list)    # run this to get the dataproc job id
+gcloud dataproc jobs wait ${JOB_ID}  # view jobs logs and wait for the job to complete
+
+# load the annotated dataset into your local elasticsearch instance
+python3 -m seqr_loading SeqrMTToESTask --local-scheduler \
+     --dest-path ${DEST_FILE} \
+     --es-host elasticsearch  \
+     --es-index ${INDEX_NAME}
```
Comment (sjahl) [inline deploy/docker/pipeline-runner/bin/load_data_dataproc.sh:58]: missing newline

File: deploy/servctl_utils/deploy_command_utils.py [hunk 3]
```diff
@@ -269,14 +269,14 @@ def deploy(deployment_target, components, output_dir=None, runtime_settings={}):
     if not components:
         raise ValueError("components list is empty")
 
-    if components:
-        run('deploy/kubectl_helpers/utils/check_context.sh {}'.format(deployment_target.replace('gcloud-', '')))
+    # if components:
```
Comment (sjahl) [inline deploy/servctl_utils/deploy_command_utils.py:272]: Were the changes in this file intentional for this PR?

**Review by sjahl — COMMENTED** [review]
The docs changes look good to me. The missing newlines in the bash scripts aren't critical, but codacy might care more than I do. Just want to confirm whether the changes to deploy_command_utils.py were desired or not.

On the subject of codacy, I agree with most of the double-quoting issues that it raised. It also flagged a markdown thing in the local_install.md file... I would be OK with trying to figure out how to get codacy to ignore markdown files in the future 😃

File: deploy/servctl_utils/deploy_command_utils.py (hunk 3 above)
Comment (hanars) [reply deploy/servctl_utils/deploy_command_utils.py]: nope, reverted these back, good catch

**Review by sjahl — APPROVED** [review]

File: deploy/docker/pipeline-runner/vep_configs/vep-GRCh37-loftee-dataproc.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+}
```
Comment (ShifaSZ) [inline deploy/docker/pipeline-runner/vep_configs/vep-GRCh37-loftee-dataproc.json:18]: Missing newline

Comment (hanars) [reply deploy/docker/pipeline-runner/vep_configs/vep-GRCh37-loftee-dataproc.json:18]: json config files are not supposed to have new lines

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**